src/main/java/com/sap/sgs/phosphor/fosstars/data/github/NvdEntryMatcher.java

```diff
@@ -49,6 +61,11 @@
    * A project to be checked.
    */
   private final GitHubProject project;
+  
+  /**
+   * A list of stop words, which should not be present when checking reference URLs.
+   */
+  private static final ArrayList<String> STOP_WORDS = new ArrayList<>(Arrays.asList("wiki"));
```

Comment:
Let's please make it simpler.

```suggestion
  private static final List<String> STOP_WORDS = Arrays.asList("wiki");
```